apps/blaze-dashboard/src/pages/setup/components/disconnected-site.jsx

```diff
@@ -45,7 +45,7 @@ export default function DisconnectedSite() {
 							<h4>{ translate( 'Connect your store' ) }</h4>
```

Comment:
```suggestion
							<h4>{ translate( 'Connect your site' ) }</h4>
```

Reply:
Nice catch. I just pushed the change in this commit: https://github.com/Automattic/wp-calypso/pull/94906/commits/54be70ccec5c35025bceffb87987e1927cb7a5a0